alfasim_sdk/hook_specs.py

```diff
@@ -44,6 +44,53 @@ def compute_source_terms(
     """
 
 
+def calculate_slip_velocity(
+    ctx: "void*",
+    U_fields: "void*",
+    alpha_f: "void*",
+    d_disp_fields: "void*",
+    P: "void*",
+    rho_f: "void*",
+    mu_f: "void*",
+    sin_theta_f: "void*",
+    delta_x_f: "void*",
+) -> "int":
+    """
+    Internal simulator hook to calculate slip velocity between fluids
+    and solids.
+
+    :param ctx: ALFAsim's plugins context
+    :param U_fields: Field velocities,
+    :param alpha_f: Field Volume fractions on faces,
+    :param d_disp_fields: Diameter of dispersed fields,
+    :param P: Pressure,
+    :param rho_f: Field densities on faces,
+    :param mu_f: Field viscosities on faces,
+    :param sin_theta_f: Sin of Theta on faces in which Theta is the angle between the Pipe and the Y-Axis,
+    :param delta_x_f: The control volume lenght related to the faces,
+
+    :returns: Return OK if successful or anything different if failed
+    """
+
+
+def calculate_slurry_viscosity(
+    ctx: "void*",
+    alpha_f: "void*",
+    mu_f: "void*",
+    mu_f_layer: "void*",
+) -> "int":
+    """
+    Internal simulator hook to calculate slurry viscosity of layer(s).
+
+    :param ctx: ALFAsim's plugins context
```

Comment:
It is not clear which of those parameters are input and which are output...

Reply:
I will add extra explanation to docstring